multimodal-search-pdf/pods/PDFcrafter.py

```diff
@@ -0,0 +1,183 @@
+import io
+from typing import Dict, List
+
+import numpy as np
+from jina.executors.segmenters import BaseSegmenter
+
+
+class PDFSegmenter(BaseSegmenter):
```

Comment:
I didn't get yhy are we copying the PDFSegmenter here? And why we call it crafter?

Reply:
Because I want to modify it. Like for text, I want to segment it into lines here.
Thanks for reminding me of that. I thought I have changed the name 😅